### PR TITLE
[DO NOT MERGE] App manifest 1.19 and devPreview October updates

### PR DESCRIPTION
**Pull request by erikadoyle**
**Note**: Requirements Targeting changes covered here: https://github.com/MicrosoftDocs/msteams-docs/pull/11018

Updates for 1.19 and devPreview manifest references:

- copilotExtensions > copilotAgents
- defaultLocalizationFile

Also updated:

- Updated loc schema reference from 1.16 to 1.19 (1.17 changes included)
- What's new (Preview, GA)
- Link to Copilot agent loc guidance from Teams loc article

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc35aa50b087e1a91341eb92ffe9de527f1329e9/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 129 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b4fbd740-fc58-4910-a245-d00a7ebc376a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cc35aa50b087e1a91341eb92ffe9de527f1329e9/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 113 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/4dd21f68-f64f-4b1e-a888-8242ccbcc11e) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33b76b9fe678ce111bdb6c9c1869e25d819c80e6/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 129 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/87e620bf-9e94-47a9-923d-c0e13149ac81) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/33b76b9fe678ce111bdb6c9c1869e25d819c80e6/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 113 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c975af23-2ca5-4c15-85cb-a7dcbec96c71) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/325a32212e180c871e85e045083d5d1b1c4d4912/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/2d662edd-65de-488d-b29d-7b5de1efcac6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/325a32212e180c871e85e045083d5d1b1c4d4912/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8fadf715-d38d-4d87-a290-104d77f9699a) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/325a32212e180c871e85e045083d5d1b1c4d4912/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 129 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/628cc4c7-809d-4de9-b779-086b12d4a179) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/325a32212e180c871e85e045083d5d1b1c4d4912/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 113 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/741822e8-13d8-4ab8-b5b4-18b162722796) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/325a32212e180c871e85e045083d5d1b1c4d4912/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/03dd60b1-1a75-4b8b-861c-591a6ee405c2) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md [hunk 1]
```diff
@@ -443,17 +445,17 @@ A color to use with and as a background for your outline icons.
...
+## copilotAgents
 
 **Optional** &ndash; Object
 
 Defines one or more agents to Microsoft 365 Copilot. [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models.
 
 |Name| Type| Maximum size | Required | Description|
 |---|---|---|---|---|
-|`declarativeCopilots`|Array of objects| 1 |✔️| Array of `declarativeCopilot` objects. |
+|`declarativeAgents`|Array of objects| 1 |✔️| Array of `declarativeAgent` objects. |
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema-dev-preview.md:456]: |`declarativeAgents`|Array of objects| 1 |✔️| Array of objects defining the declarative agent. |

File: msteams-platform/resources/schema/manifest-schema.md [hunk 2]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
 
 **Optional** &ndash; Object
 
-Allows the specification of a default language and provides pointers to more language files. For more information, see [localization](~/concepts/build-and-test/apps-localization.md).
+Allows the specification of a default language, and pointers to additional language files. See [localization](~/concepts/build-and-test/apps-localization.md).
 
 |Name| Maximum size | Required | Description|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:436]:
```Suggestion
|Name| Type | Maximum size | Required | Description|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 3]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
...
 |---|---|---|---|
-|`defaultLanguageTag`||✔️|The language tag of the strings in this top-level app manifest file.|
+|`defaultLanguageTag`||✔️|The language tag for the strings in this app manifest file. For example, `en`.|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:438]:
```Suggestion
|`defaultLanguageTag`| String | |✔️|The language tag for the strings in this app manifest file. For example, `en`|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 4]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
...
+|`defaultLanguageFile`|2048 characters|| A relative file path to the .json file that contains the strings. If unspecified, strings are taken directly from the app manifest file. A default language file is required for [Copilot agents that support multiple languages](/microsoft-365-copilot/extensibility/agents-are-apps#localizing-your-agent).|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:439]:
```Suggestion
|`defaultLanguageFile`| String | 2048 characters|| A relative file path to the .json file that contains the strings. If unspecified, strings are taken directly from the app manifest file. A default language file is required for [Copilot agents that support multiple languages](/microsoft-365-copilot/extensibility/agents-are-apps#localizing-your-agent).|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 5]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
...
 ### localizationInfo.additionalLanguages
 
-An array of objects specifying more language translations.
+An array of objects with the following properties to specify additional language translations.
 
-|Name| Type | Maximum size | Required | Description|
-|---|---|---|---|---|
-|`languageTag`|String||✔️|The language tag of the strings in the provided file.|
-|`file`|String|2048 characters|✔️|A relative file path to the .json file containing the translated strings.|
+|Name| Maximum size | Required | Description|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:445]:
```Suggestion
|Name| Type | Maximum size | Required | Description|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 6]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
...
+|---|---|---|---|
+|`languageTag`||✔️|The language tag of the strings in the provided file. For example, `es`.|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:447]:
```Suggestion
|`languageTag`| String | |✔️|The language tag of the strings in the provided file. For example, `es`|
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 7]
```diff
@@ -422,20 +431,21 @@ Ensure that your description describes your experience and helps potential custo
...
+|`file`|2048 characters|✔️|A relative file path to the .json file that contains the translated strings.|
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:448]:
```Suggestion
|`file`| String | 2048 characters|✔️|A relative file path to the .json file that contains the translated strings.|

File: msteams-platform/resources/schema/manifest-schema.md [hunk 8]
```diff
@@ -456,6 +466,25 @@ A color to use and as a background for your color icons.
...
 
+## copilotAgents
+
+**Optional** &ndash; Object
+
+Defines one or more agents to Microsoft 365 Copilot. [Declarative agents](/microsoft-365-copilot/extensibility/overview-declarative-agent) are customizations of Microsoft 365 Copilot that run on its same orchestrator and foundation models.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`declarativeAgents`|Array of objects| 1 |✔️| Array of `declarativeAgent` objects. |
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:477]:
```Suggestion
|`declarativeAgents`|Array of objects| 1 |✔️| Array of objects defining the declarative agent. |
```

File: msteams-platform/resources/schema/manifest-schema.md [hunk 9]
```diff
@@ -456,6 +466,25 @@ A color to use and as a background for your color icons.
...
+
+### declarativeAgent
+
+Represents a customization of Microsoft 365 Copilot, as defined by its manifest file.
+
+|Name| Type| Maximum size | Required | Description|
+|---|---|---|---|---|
+|`id`|String| |✔️| Unique identifier for the agent. When using Microsoft Copilot Studio to build agents, this is auto-generated. Otherwise, manually assign the value according to your own conventions or preference. |
+|`file`|String| |✔️| Relative path within the app package to the [declarative agent manifest](/microsoft-365-copilot/extensibility/declarative-agent-manifest) file. |
```
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:486]:
```Suggestion
|`file`| String | 2048 characters |✔️| Relative file path within the app package to the [declarative agent manifest](/microsoft-365-copilot/extensibility/declarative-agent-manifest) file. |
```

File: msteams-platform/resources/schema/manifest-schema.md (hunk 3 above)
Comment (v-ramyas5) [inline msteams-platform/resources/schema/manifest-schema.md:438]: In the schema, seems the default value is `en-us`. Will that affect the new change? Do we need to check with the schema PM for this? or mention the default value in the doc?
image

**Review by v-ramyas5 — CHANGES_REQUESTED** [review]
Hi @erikadoyle,
Have shared some small suggestions. Please accommodate.
Since, we have decided to add the 'Type' column in the property table even if all the properties are strings to make it clear to the developers and to follow consistency, have added that also as a suggestion.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c89eac24bc830d1f8d99338b25d7cacc19be91/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6c4fc593-fd80-4ca9-94cb-cc4eea757776) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c89eac24bc830d1f8d99338b25d7cacc19be91/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8c7b427a-7216-4f87-aae4-b953287aa7e4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c89eac24bc830d1f8d99338b25d7cacc19be91/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f1b45d00-e27b-4004-a238-ab4b084234a4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c89eac24bc830d1f8d99338b25d7cacc19be91/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 114 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/eaf011ea-e1a8-4aab-ab7e-0c3113599639) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d1c89eac24bc830d1f8d99338b25d7cacc19be91/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/be9c41d0-3260-459c-a8d3-c96118c3b2aa) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema-dev-preview.md (hunk 1 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema-dev-preview.md:459]: Added

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85b75696244f16f817c02143ef0dc7980fc13328/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/adc6e1f2-86b0-47d9-bd13-5881fea4473d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85b75696244f16f817c02143ef0dc7980fc13328/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1339e362-05e9-4371-97e3-d0bd709681aa) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85b75696244f16f817c02143ef0dc7980fc13328/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/05df5c7d-81b4-467a-af64-2654173d4f15) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85b75696244f16f817c02143ef0dc7980fc13328/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 114 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/845e1f91-ae52-4501-8bab-e49f287d8d66) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/85b75696244f16f817c02143ef0dc7980fc13328/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a75d16ec-5b4a-42f7-9270-1c8bc317fe74) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/resources/schema/manifest-schema.md (hunk 3 above)
Comment (erikadoyle) [reply msteams-platform/resources/schema/manifest-schema.md]: Ah, good catch! I've incorporated the default value to description.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e900bed100b9a078f534aa66c4cade35ac4f332d/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6d40151d-4616-4f80-b7b2-6274ac6ae69e) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e900bed100b9a078f534aa66c4cade35ac4f332d/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e9320620-a312-45fb-8877-5f27d787c4cb) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e900bed100b9a078f534aa66c4cade35ac4f332d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/91cf3dc9-7b82-41ce-8cbd-3753762dee7f) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e900bed100b9a078f534aa66c4cade35ac4f332d/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 114 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/374051fd-07ea-4006-9385-eede3c835ae6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/e900bed100b9a078f534aa66c4cade35ac4f332d/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/3006547b-dc19-461d-bbb4-ab08b2eda862) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by erikadoyle** [comment]
Thank you very much @v-ramyas5! I've incorporated your suggestions.

**Comment by v-ramyas5** [comment]
> Thank you very much @v-ramyas5! I've incorporated your suggestions.

Thank you @erikadoyle for incorporating the changes.

**Review by v-ramyas5 — DISMISSED** [review]

**Review by Wajeed-msft — DISMISSED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9490c6afaeadb1485c1b2be7b3c258a531c6c534/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e786b99e-a51a-487e-ace5-49dd70f19b33) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9490c6afaeadb1485c1b2be7b3c258a531c6c534/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a995032d-8612-4a80-91fa-6d5db383e64d) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9490c6afaeadb1485c1b2be7b3c258a531c6c534/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/e4d1a120-9e7a-4d51-a254-fae4e78c7fa4) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9490c6afaeadb1485c1b2be7b3c258a531c6c534/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b0463d85-b18e-48af-b059-4870bee1083c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/9490c6afaeadb1485c1b2be7b3c258a531c6c534/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/b27df785-3649-48ad-af89-aaa9fdcdb7b0) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/concepts/build-and-test/apps-localization.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c672e4fc36d65d923553a95ad1770f7da857f1f/msteams-platform%2Fconcepts%2Fbuild-and-test%2Fapps-localization.md) | 88 | 11 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/c638f8e3-0b07-4d5d-b48a-7dd81dd3ad26) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/localization-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c672e4fc36d65d923553a95ad1770f7da857f1f/msteams-platform%2Fresources%2Fschema%2Flocalization-schema.md) | 99 | 4 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/89e0fe50-f8ec-4bed-9dec-a8ee5d00dd30) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema-dev-preview.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c672e4fc36d65d923553a95ad1770f7da857f1f/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema-dev-preview.md) | 85 | 130 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62cb436c-edb7-4202-916a-8913880766f6) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/resources/schema/manifest-schema.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c672e4fc36d65d923553a95ad1770f7da857f1f/msteams-platform%2Fresources%2Fschema%2Fmanifest-schema.md) | 86 | 115 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8a3dcc6a-4742-41ac-bb8f-fe64b407b75c) | [:white_check_mark:](# "Successfully processed") |
| [msteams-platform/whats-new.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3c672e4fc36d65d923553a95ad1770f7da857f1f/msteams-platform%2Fwhats-new.md) | 90 | 91 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1476c696-37d6-44c0-8a3f-c866b356ba03) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by erikadoyle** [comment]
This PR is now obsolete. New 1.19 docs playload is here: https://github.com/MicrosoftDocs/msteams-docs/pull/11697

**Pull request closed without merging**